packages/examples/packages/network-access/src/index.test.ts

```diff
@@ -1,8 +1,10 @@
 import { expect } from '@jest/globals';
-import { NodeProcessExecutionService } from '@metamask/snaps-controllers';
+import { NodeProcessExecutionService } from '@metamask/snaps-controllers/node';
```

Comment:
why'd the import change?

Reply:
Because NodeProcessExecutionService isn't exported by normal snap-controllers